pymess/backend/dialer/daktela.py

```diff
@@ -0,0 +1,100 @@
+from chamber.utils.datastructures import ChoicesEnum
+from django.utils import timezone
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _l
+
+from pymess.backend.dialer import DialerBackend
+from pymess.config import settings
+from pymess.models import DialerMessage
+from pymess.utils.logged_requests import generate_session
+
+
+class DaktelaDialerBackend(DialerBackend):
+    """
+    Dialer backend implementing Daktela service https://www.daktela.com/api/v6/models/campaignsrecords
+    """
+
+    STATE = ChoicesEnum(
+        ('NOT_ASSIGNED', _l('not assigned'), 'not assigned'),
+        ('READY', _l('ready'), 'ready'),
+        ('RESCHEDULED_BY_DIALER', _l('rescheduled by dialer'), 'rescheduled by dialer'),
+        ('CALL_IN_PROGRESS', _l('call in progress'), 'call in progress'),
+        ('HANGUP', _l('hangup'), 'hangup'),
+        ('DONE', _l('done'), 'done'),
+        ('RESCHEDULED', _l('rescheduled'), 'rescheduled'),
+    )
+
+    # TODO: clean-up mappings
+    STATES_MAPPING = {
+        STATE.NOT_ASSIGNED: DialerMessage.STATE.UNKNOWN,
+        STATE.READY: DialerMessage.STATE.READY,
+        STATE.RESCHEDULED_BY_DIALER: DialerMessage.STATE.RESCHEDULED,
+        STATE.CALL_IN_PROGRESS: DialerMessage.STATE.IN_PROGRESS,
+        STATE.HANGUP: DialerMessage.STATE.DONE,
+        STATE.DONE: DialerMessage.STATE.DONE,
+        STATE.RESCHEDULED: DialerMessage.STATE.RESCHEDULED,
+    }
+
+    @staticmethod
+    def _get_dialer_api_url(name=None):
+        base_url = '/'.join((
+            settings.DIALER_API_URL,
+            settings.DIALER_API_VERSION,
+            settings.DIALER_API_RESOURCE,
```

Comment:
nestačilo by to dát do jednoho configu?

Reply:
ako to myslis?

Reply:
teoreticky to mozem zgrcnut do jedineho stringu. chcel som to spravit trosku flexibilnejsie. v zasade toto je asi jedno. ak trvate obaja na tom, aby to bol jeden pevny string, kotry v pripade zmeny cohokolvek bude treba opatrne menit, tak to tak spravim

Reply:
ano dej to do jednoho

Reply:
OK